src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/FilterMutectCalls.java

```diff
@@ -41,24 +46,33 @@
  *
  * <p>
  *     If given a --contaminationTable file, e.g. results from
- *     {@link org.broadinstitute.hellbender.tools.walkers.contamination.CalculateContamination}, the tool will additionally
+ *     {@link CalculateContamination}, the tool will additionally
```

Comment:
- Kebabbify --contaminationTable , --contamination_fraction_to_filter etc.
- Kebabbify or omit `--javaOptions "-Xmx4g"`

Same elsewhere.

Reply:
done except `--contamination_fraction_to_filter` is in `StandardCallerArgumentCollection` and I assume is someone else's responsibility.

Reply:
Nope, I missed those.  Hopefully @jamesemery will put that into #3891